templates/config/controller/deployment.yaml.tpl

```
@@ -37,6 +37,10 @@ spec:
         - "$(ACK_RESOURCE_TAGS)"
         - --watch-namespace
         - "$(ACK_WATCH_NAMESPACE)"
+        - --enable-leader-election
+        - "$(ENABLE_LEADER_ELECTION)"
```

These 2 lines need to be combined, `booleans` need to be on the same line, since the presence of the flag is a truthy statement, and nothing else gets parsed after that. This is coming from: 

```
Package pflag is a drop-in replacement for Go's flag package, implementing
POSIX/GNU-style --flags.

pflag is compatible with the GNU extensions to the POSIX recommendations
for command-line options. See
http://www.gnu.org/software/libc/manual/html_node/Argument-Syntax.html
```

Our existing boolean flags are also incorrect, I'll raise a PR.
```suggestion
        - --enable-leader-election=$(ENABLE_LEADER_ELECTION)
```

---

Addressed